lib/src/camera.dart

```diff
@@ -0,0 +1,45 @@
+import "package:burt_network/burt_network.dart";
+import "package:opencv_ffi/opencv_ffi.dart";
+import "dart:async";
+import "collection.dart";
+
+class CameraManager {
+  late final Camera camera;
+  late final CameraDetails details;
+  Timer? timer;
+  
+  CameraManager({required this.camera, required this.details}){ 
+    init();
+  }
+
+  void init(){  // init the timer 
+    timer = Timer.periodic(Duration(milliseconds: 1000 ~/ details.fps), (runner) {
+      if(!camera.isOpened){
+        runner.cancel();
+        details.mergeFromMessage(CameraDetails(status: CameraStatus.CAMERA_DISCONNECTED));
+      }
+      sendFrame();
+    });
```

Comment:
Maybe make this just call `sendFrame`, which can handle checking the camera. Truth be told though, if the camera isn't opened, the read will fail. The way we did this in Python is check once for not connected, then if it _was_ connected and no longer reads, it's `NOT_RESPONDING`. That way you don't have to check its connection every frame (could be 60 FPS for 5x cameras)